xUnitRevitUtils2021/xru.cs

```diff
@@ -106,7 +106,7 @@ public static Document CreateNewDoc(string templatePath, string filePath, bool o
     /// <param name="doc">Revit Document</param>
     /// <param name="transactionName">Transaction Name</param>
     /// <returns></returns>
-    public static Task RunInTransaction(Action action, Document doc, string transactionName = "transaction")
+    public static Task RunInTransaction(Action action, Document doc, string transactionName = "transaction", bool ignoreWarnings = false)
```

Comment:
Looks great, would you mind also adding a description for the `ignoreWarnings` input in the comment section?
Thanks!

Reply:
Resolved in latest commit